### DISCUSSION TITLE
Does search_all_tweets function provide replies to a tweet?

### DISCUSSION THREAD

**Discussion by gabett** [tweepy/tweepy#2066] [Questions] [answered]
Hi everyone,

I have a question regarging the obtainment of tweet's replies using the _search_all_tweets_ function in combination with Paginator.

Let's say I have something like:

`            
pages = tweepy.Paginator(self.client.search_all_tweets,   
            query = f"conversation_id:{conversation_id}",   
            tweet_fields = self.fields['tweet_fields'],  
            user_fields = self.fields['user_fields'],  
            place_fields = self.fields['place_fields'],  
            media_fields = self.fields['media_fields'],  
            expansions = self.expansions,  
            
)`

Assuming that all fields and connection are correct, it looks like tweepy always returns an empty result. So, the question is: how do you get replies to a tweet using academical APIs?

**Comment by Harmon758** [comment, marked as answer]
Assuming all the parameters are actually valid, this should work.
You'll need to provide a reproducible example (MCVE/SSCCE) of what you're doing for anyone to be able to help you.

For code block usage, see https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks.
